### ISSUE TITLE
Can't use the provider - documentation improvement required?

### ISSUE THREAD

**Issue by tlm-daniele-ricci** [bruxisma/terraform-provider-env#19]
Hi all,

I'm trying to use the provider without success: I added

```
provider "env" {}
```

but I get following error:

```
Error: Failed to query available provider packages

Could not retrieve the list of available versions for provider
hashicorp/env: provider registry registry.terraform.io does not have a
provider named registry.terraform.io/hashicorp/env
```

What am I missing?
Thank you

**Comment by bruxisma** [comment]
Hi! Thanks for your interest in my small project 😅

This plugin hasn't been published to the terraform registry yet as I want to have a secure and also hands off approach to publishing and that requires a way to manage the GPG key without too much concern on my part.

I also recently changed my username on here and held off on getting back to this because I've started to look more into [pulumi](https://www.pulumi.com/) as it has a (in my personal opinion) better approach for reading environment variables and storing encrypted configuration values.

I currently work a 9-5 and don't know when I'll have time to get this project published (if ever). However, feel free to fork + publish yourself if you're ok with that 🙂
